Applications/WebPortal/WebApp/library/source/pages/account_directory_page/account_directory_page.tsx

```diff
@@ -85,7 +85,9 @@ export class AccountDirectoryPage extends React.Component<Properties, State> {
           accounts={accounts}
           filter={this.state.filter}
           isOpen={this.state.openedGroups.test(group)}
-          onClick={this.onCardClick}/>);
+          onAccountClick={this.props.onNewAccountClick}
```

Comment:
```onNewAccountClick``` is called when the New Account button is clicked, not when the GroupCard is clicked.